published/012-2018-Oct-2.md

```diff
@@ -0,0 +1,63 @@
++ [IPFS & Libp2p Developer Meetings - Growing The Decentralized Tech That Helps OpenBazaar Work](https://openbazaar.org/blog/IPFS-Libp2p-Developer-Meetings-Growing-The-Decentralized-Tech-That-Helps-OpenBazaar-Work/), OpenBazaar, 30 Jul 2018
+
+## Updates and new releases
+*See the latest releases of IPFS tools and projects across the ecosystem.*
+
++ [ipld-explorer-cli 0.14](https://blog.ipfs.io/44-ipld-explorer-cli-0-14/) - IPLD (Inter Planetary Linked Data) is the underlying data structure used by IPFS that allows it to store, retrieve and traverse through any type of content-addressed data.
++ [ipfs-cluster 0.6.0](https://github.com/ipfs/ipfs-cluster/blob/0.6.0/changelog/CHANGELOG.md) – Increased the minor release number to signal changes to the Go APIs after upgrading to the new cid package, but, other than that, this release does not include any major changes.
++ [js-ipfs 0.32.3](https://github.com/ipfs/js-ipfs/releases/tag/v0.32.3) – Some bug fixes for the JavaScript implementation of the IPFS protocol.
++ [qri-io 0.4.0](https://github.com/qri-io/frontend/releases/tag/v0.4.0) - qri electron & web frontend, react & redux webapp for qri.io, plus [check out the screencast demo](https://www.youtube.com/watch?v=3CSPA3qGU5g)!
++ [Announcing the launch of 0x protocol v2.0!](https://blog.0xproject.com/0x-protocol-v2-0-is-live-183aac180149?gi=3d08c2ad1aa2)
+
+## Tools and projects we <3
+*[Awesome IPFS](https://awesome.ipfs.io/) is a community maintained and updated list of projects, tools, or pretty much any things related to IPFS that are totally awesome. To see more, or add yours to the list, visit [Awesome IPFS on GitHub](https://github.com/ipfs/awesome-ipfs).* 
+
++ [Download the Brave Beta](https://brave.com/download-beta/) and get an early preview for new versions of Brave.
++ [Announcing Hyperswarm](https://pfrazee.hashbase.io/blog/hyperswarm), Paul Frazee, 26 Sep 2018
++ [Chattervox](https://github.com/brannondorsey/chattervox), 📡 An AX.25 packet radio chat protocol with support for digital signatures and binary compression. Like IRC over radio waves. 〰 24 Sep 2018
++ [Agoric Releases Distributed Smart Contract Proof of Concept](https://agoric.com/proof-of-concept-release/), 20 Sep 2018
++ This [0x v2.0 AMA on Reddit](https://www.reddit.com/r/0xProject/comments/9itlkl/the_0x_v20_ama_wednesday_september_26th_2018_10/)
++ [Arbore](https://arbo.re/) is a free and open-source file sharing application that enables you to send your pictures, documents, files to your contacts privately and without limits.
++ [Creativechain](https://www.creativechain.org/project/) – A tool for the empowerment of artists and all kinds of creative people who produce, share and distribute digital content.
++ [Manyverse](https://www.manyver.se/) A solid network off the grid, download the beta version for Android. 
```

Comment:
_a **social** network_. Not to be confused with [solid](https://solid.mit.edu/); the Manyverse is built on [secure-scuttlebutt](https://www.scuttlebutt.nz/) and it's the first really polished mobile client for that network.

Reply:
Thanks @olizilla!